omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Omics Orientation
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** None
+
+**Learning Objectives**
+
+@learning_objectives
+
+
+</div>
+
+
+## What is "Omics"?
+
+<div class = "learnmore">
+Much of the content of this module, particularly the definitions of specific omics domains, was derived from an article by Hasin *et al.* published in the journal *Genome Biology*. The citation is below and the article is open source if you are interesting in reading more about this topic.
+
+<div style = "margin-left: 40px; text-indent: -40px; font-size:0.8em;">
+
+Hasin, Y., Seldin, M. & Lusis, A. Multi-omics approaches to disease. Genome Biol 18, 83 (2017). https://doi.org/10.1186/s13059-017-1215-1
+
+</div>
+
+</div>
+
+**Omics** is a suffix added to a large number of biological disciplines which denotes a "comprehensive, or global, assessment of a set of molecules" (Hasin et al., 2017) related to that domain.  Genomics is possibly the most well-known member of this family of disciplines, but is certainly not the only one! Omics has become something of a buzzword in recent years with the improvement of high-throughput technologies and increased ability to analyze large amounts of data.
+
+### Why study omics?
+
+While studying single molecules or genetic loci is often valuable, biological processes and diseases are generally complicated and are rarely explained by variation in just one component of a system. For example, some genetic diseases and disorders are polygenic (due at least in part to contributions from multiple genes), and many are believed to result from changes in gene regulation rather than changes in the genes themselves. Furthermore, these variations contribute to individual phenotypes in various ways; genetic background and environment play their own roles! In short, there are a lot of moving parts, and studying omics can give researchers a "global" view of biology. This can help explain such things as why some variants cause disease in some individuals and not others, or how the components of a biological system work together to form a whole.
+
+### Quiz: What is omics?
+
+True or False: Omics gives you a narrow, simplified view of biological systems.
+
+[( )] TRUE
+[(X)] FALSE
+***
+<div class = "answer">
+While there is often value in simplifying the study of biological phenomena to one or a few specific targets, omics fields don't take this approach; rather, omics fields leverage the power of big data to try to capture the complexity of biological systems and processes.  
+</div>
+***
+
+## Omics topics
+
+There are many "omics" topics that correspond to various biological domains. The following sections will discuss some of the most popular topics: genomics, transcriptomics, and metabolomics.
```

Comment:
The topics in the table of contents are Genomics, Transcriptomics, Proteomics, and Other topic/Multi-omics. Should metabolomics have its own subsection? Or should this have proteomics instead of metabolomics?

Reply:
Oh whoops, that's a holdover from an easier version, will change!